### DISCUSSION TITLE
Root cells are too dispersed

### DISCUSSION THREAD

**Discussion by MatsuSui** [theislab/scvelo#890] [Q&A]
Is there any method can gather root cells into 1 or 2 regions? I get 9  oot cell regions and 1 endpoint regions after performing scvelo.
![Image](https://user-images.githubusercontent.com/102005180/174430964-a5195f22-1633-4fca-867a-3f7f1857d7c5.png)
![Image](https://user-images.githubusercontent.com/102005180/174430970-f4b45a20-c632-441e-8a88-903142501d6a.png)

**Comment by WeilerP** [comment]
@MatsuSui, I suggest you use [CellRank](https://cellrank.readthedocs.io/en/stable/) for inferring initial and terminal states. It was developed precisely with these kinds of applications in mind.
